kernel/common/get_slice.rs

```diff
@@ -1,8 +1,6 @@
-use core::slice;
-use core::str;
 use collections::range::RangeArgument;
 use core::ops::Range;
-use core::cmp::{max, min};
+use core::cmp::min;
```

Comment:
Nope, my changes haven't been merged to master yet. I'd think @jackpot51 would need to merge what we have in the `orbital` branch to `master` in order to avoid this conflict. Also your PR is pointing to be merged with `master` as well, if that was indented.